qiskit/passmanager/passmanager.py

```diff
@@ -350,4 +350,5 @@ def _run_workflow_in_new_process(
         program=program,
         pass_manager=dill.loads(pass_manager_bin),
         initial_property_set=initial_property_set,
+        callback=callback,
```

Comment:
Once serialized, the `callback` argument should be deserialized before calling to `_run_workflow`.